manifests/letsencrypt-issuer.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if eq .dns.type "route53" }}
```

Comment:
We try and avoid templating entire documents, so in this case it would be better to move this to install.go